packages/python-packages/apiview-copilot/src/agent/_agent.py

```diff
@@ -0,0 +1,67 @@
+from contextlib import asynccontextmanager
+from dotenv import load_dotenv
+from azure.identity.aio import DefaultAzureCredential
+from semantic_kernel.agents import AzureAIAgent, AzureAIAgentSettings, RunPollingOptions
+from datetime import timedelta
+import logging
+import os
+
+from .plugins import SearchPlugin, UtilityPlugin, ApiReviewPlugin, DatabasePlugin
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+
+load_dotenv(override=True)
+
+
+@asynccontextmanager
+async def get_main_agent():
+    ai_agent_settings = AzureAIAgentSettings(
+        endpoint=os.getenv("AZURE_AI_AGENT_ENDPOINT"),
+        model_deployment_name=os.getenv("AZURE_AI_AGENT_MODEL_DEPLOYMENT_NAME"),
+        api_version=os.getenv("AZURE_AI_AGENT_API_VERSION"),
+    )
+    ai_instructions = """
+Your job is to receive a request from the user, determine their intent, and pass the request to the
+appropriate agent or agents for processing. You will then return the response from that agent to the user.
+If there's no agent that can handle the request, you will respond with a message indicating that you cannot
+process the request.
+exit
```

Comment:
Is this `exit` a typo?

Reply:
Yeah probably :P  You type "exit" or "quit" to end the agent chat, so I probably was trying to do that with the wrong window in focus.

Reply:
Fixed.